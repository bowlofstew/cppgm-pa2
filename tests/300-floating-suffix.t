42.421e20f 42.525e-20F
52.521515e300 421.45252e-300
2.52e4000l 22.5e-4000l

