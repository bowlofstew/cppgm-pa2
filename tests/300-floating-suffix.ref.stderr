literal 42.421e20f float F7F66563
literal 42.525e-20F float F705FB20
literal 52.521515e300 double 78309204489B937E
literal 421.45252e-300 double 3A712B68E6A33102
literal 2.52e4000l long double D77E4F8ED1052184E87391DBFF7F0000
literal 22.5e-4000l long double 16C866DF746AB6DB1B0C91DBFF7F0000
eof
